Makefile

```
@@ -32,7 +32,7 @@ all: build
 
 # Controller-gen version
 # f284e2e8... is master ahead of v0.5.0 which has ability to generate embedded objectmeta in CRDs
-CONTROLLER_GEN_VERSION=v0.6.2
+CONTROLLER_GEN_VERSION=v0.11.3
```

```suggestion
CONTROLLER_GEN_VERSION=v0.11.3
```
can we do the controller runtime changes in the different pr?
since this are not related to k8s 1.26, right?

---

Actually they are too dependent,
We need min  v0.11.0 https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.11.0 to support it